### ISSUE TITLE
IDC: Handle light color schemes in admin bar menu

### ISSUE THREAD

**Issue by ebinnion** [Automattic/jetpack#5515]
After #5509, the admin bar menu item for IDC looks like this on a light scheme:

<img width="1093" alt="screen shot 2016-11-04 at 12 06 31 pm" src="https://cloud.githubusercontent.com/assets/1126811/20014982/56e797a4-a287-11e6-97d0-0749ee431787.png">

It's low priority, but we could try to have a dark and light scheme for our menu item that is loaded conditionally. So, when the admin color scheme is light, we go dark. ¯\_(ツ)_/¯

**Comment by samiff** [comment]
Cleaning stale issues 🧹  The menu item style was eventually updated (orange background), which is probably ideal so users are aware of an IDC and resolve it ASAP.

![Screenshot 2023-07-13 at 1 36 16 PM](https://github.com/Automattic/jetpack/assets/15803018/24ed9825-2e17-4fcc-9a21-7dac5db99e76)
